src/projectscene/qml/Audacity/ProjectScene/clipsview/ClipItem.qml

```diff
@@ -474,15 +474,21 @@ Rectangle {
                     height: 16
                     anchors.verticalCenter: parent.verticalCenter
 
-                    menuModel: singleClipContextMenuModel
+                    menuModel: (root.isMultiSelectionActive && root.clipSelected) ? multiClipContextMenuModel : singleClipContextMenuModel
 
                     onHandleMenuItem: function(itemId) {
-                        Qt.callLater(singleClipContextMenuModel.handleMenuItem, itemId)
+                        if (root.isMultiSelectionActive && root.clipSelected) {
+                            Qt.callLater(multiClipContextMenuModel.handleMenuItem, itemId)
+                        } else {
+                            Qt.callLater(singleClipContextMenuModel.handleMenuItem, itemId)
+                        }
                     }
 
                     onClicked: {
-                        root.requestSelectionReset()
-                        root.requestSelected()
+                        if (!(root.isMultiSelectionActive && root.clipSelected)) {
```

Comment:
you can create a property for `root.isMultiSelectionActive && root.clipSelected` and reuse it